code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -166,7 +166,7 @@
 	icon_state = "carp"
 
 /obj/item/weapon/storage/fancy/cigarettes/cigpack_syndicate
-	name = "cigarette packet"
+	name = "\improper Syndicig cigarette packet"

```

Comment:
The point of them having no brand name is because they are a traitor item. It's like calling the AI detector "AI detector" instead of "multitool".